WalletWasabi/Blockchain/Analysis/BlockchainAnalyzer.cs

```diff
@@ -135,10 +135,11 @@ private void AnalyzeCoinjoinWalletOutputs(SmartTransaction tx, double startingMi
 			// Picking randomly an output would make our anonset: total/ours.
 			double anonymityGain = Math.Min(CoinjoinAnalyzer.ComputeAnonymityContribution(virtualOutput.Coins.First()), foreignInputCount);
```

Comment:
For better readability consider moving this code down where `anonymityGain` is used.

Reply:
Done in https://github.com/zkSNACKs/WalletWasabi/pull/10100/commits/37af240ef13132f51aad037c6734c6c9cd1c2b81.